doc/nrf/gs_recommended_versions.rst

```diff
@@ -22,6 +22,73 @@ However, there are some Zephyr features that are currently only available on Lin
 
    Before you start setting up the toolchain, install available updates for your operating system.
 
+.. _gs_supported_OS:
+
+Supported operating systems
+***************************
+
+The following table shows the operating systems and their various versions that provide support for the |NCS| tools:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operating System
+     - x86
+     - x64
+     - ARM64
+   * - `Windows 11`_
+     - Tier 3
```

Comment:
We need to verify with TPM, if this is correct. It differs from the requirements.